package.json

```diff
@@ -225,7 +226,7 @@
     "failFast": false,
     "failWithoutAssertions": false,
     "tap": false,
-    "timeout": "2m"
+    "timeout": "5m"
```

Comment:
A higher timeout allows us to see why the test is taking a long time to execute (instead of `ava` killing it)